src/src/version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Version file for displaying the respective version of 
+the tools and online-tools repositories.
+"""
+
+spdx_version = '1.0'
+
+"""
+Visit https://github.com/spdx/tools/releases to know about the tools releases.
+"""
+java_tools_version = '2.1.20'
```

Comment:
The current tar is not updated with the latest spdx tools release. I think it is 2.1.8 instead (check the last commit on the tar)